src/Storages/ExternalStream/Kafka/KafkaSink.h

```diff
@@ -100,6 +101,7 @@ class KafkaSink final : public SinkToStorage
     Int32 partition_cnt {0};
     bool one_message_per_row {false};
     Int32 topic_refresh_interval_ms = 0;
+    UInt64 max_message_batch_size = 0;
```

Comment:
we probably don't need save this as a data member since it got passed to `WriteBufferFromKafkaSink` and `pending_data` ?

Reply:
Yes, it does not need to be. Will remove it.